samples/snippets/deletes/deletes_snippets.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
+
+# Copyright 2022, Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from google.cloud import bigtable
+
+# Write your code here.
+
+
+# [START bigtable_delete_from_column_sample]
+def delete_from_column_sample(project_id, instance_id, table_id):
+    client = bigtable.Client(project=project_id, admin=True)
+    instance = client.instance(instance_id)
+    table = instance.table(table_id)
+    row = table.row("phone#4c410523#20190501")
+    row.delete_cell(column_family_id="cell_plan", column="data_plan_01gb")
+    row.commit()
+
+
+# [END bigtable_delete_from_column_sample]
+
+# [START bigtable_delete_from_column_family_sample]
+def delete_from_column_family_sample(project_id, instance_id, table_id):
+    client = bigtable.Client(project=project_id, admin=True)
+    instance = client.instance(instance_id)
+    table = instance.table(table_id)
+    row = table.row("phone#4c410523#20190501")
+    row.delete_cells(
+        column_family_id="cell_plan", columns=["data_plan_01gb", "data_plan_05gb"]
```

Comment:
do we need to specify the columns here, or is just the column family id sufficient?

Reply:
It's needed because `columns` is a required parameter in `delete_cells`
https://googleapis.dev/python/bigtable/latest/row.html?highlight=delete_cells#google.cloud.bigtable.row.ConditionalRow.delete_cells